### ISSUE TITLE
Create unit tests for RcDropdown components

### ISSUE THREAD

**Issue by cnotv** [rancher/dashboard#14035]
# Description

The new [`RcDropdown` components](https://github.com/rancher/dashboard/tree/master/pkg/rancher-components/src/components/RcDropdown) lack unit tests and should be added.

Adding `area/accessibility` label only for track purposes, but this is extended to any use case.

Note: Crossed while working on https://github.com/rancher/dashboard/issues/4152

**Comment by rak-phillip** [comment]
Unit tests added in #15012
